python/test/gluon/test_frontend.py

```diff
@@ -2997,6 +2997,27 @@ def kernel(bf16_ptr):
 """)
 
 
+@pytest.mark.parametrize("target", [HIP_TARGET_CDNA3, HIP_TARGET_CDNA4, HIP_TARGET_GFX1250])
+def test_amd_warp_pipeline(target):
+
+    @gluon.jit
+    def kernel():
+        c0: ttgl.constexpr = 0
+        one: ttgl.constexpr = 1
+
+        # Simple loop with an explicit split point
+        for i in range(c0, 10, one):
+            with ttgl.amd.warp_pipeline_stage("stage0"):
```

Comment:
IIUC, these "stage0" and "stage1" is just informational without being semantic bearing?